lib/alephant/storage.rb

```diff
@@ -79,5 +94,11 @@ def add_custom_meta(object)
         :"head_Last-Modified" => DateTime.parse(object.last_modified.to_s).httpdate
       }
     end
+
+    def client
+      options = {}
+      options.merge!({endpoint: ENV['AWS_S3_ENDPOINT']}) if ENV['AWS_S3_ENDPOINT']
```

Comment:
Is there a specific reason for the `merge`?
`options = ENV['AWS_S3_ENDPOINT'] ? {endpoint: ENV['AWS_S3_ENDPOINT']} : {}` perhaps?
or if you still want it separated out
```
options = {}
options[:endpoint] = ENV['AWS_S3_ENDPOINT'] if ENV['AWS_S3_ENDPOINT']
```

Reply:
Also we stub this method out in the tests, so it's not tested.